inductiva/resources/machines_base.py

```diff
@@ -85,8 +84,9 @@ def _register_machine_group(self, **kwargs):
             resp = self._api.register_vm_group(body=instance_group_config)
         except (exceptions.ApiValueError, exceptions.ApiException) as e:
             logs.log_and_exit(logging.getLogger(), logging.ERROR,
-                              "Resource registering failed with exception %s",
-                              e)
+                              "Registering machine group failed"\
+                              " with exception %s", e,
```

Comment:
don't we have a way of keeping this lines together?

```python
logs.log_and_exit(
        logging.getLogger(), logging.ERROR,
        "Registering machine group failed with exception %s",
        e, exc_info=e)
```

Reply:
Done